ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml

```diff
@@ -1,4 +1,140 @@
 periodics:
+- agent: kubernetes
+  cluster: api.ci
+  decorate: true
+  interval: 48h
+  labels:
+    ci.openshift.io/release-type: informing
+    job-release: "4.5"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: release-openshift-origin-installer-cert-recovery-4.5
+  spec:
+    containers:
+    - args:
+      - --artifact-dir=$(ARTIFACTS)
+      - --give-pr-author-access-to-namespace=true
+      - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+      - --input-hash=$(BUILD_ID)
+      - --input-hash=$(JOB_NAME)
+      - --kubeconfig=/etc/apici/kubeconfig
+      - --secret-dir=/usr/local/libvirt-31-day-cluster-profile
+      - --secret-dir=/etc/pull-secret
+      - --target=libvirt-31-day
+      - --template=/usr/local/libvirt-31-day
+      command:
+      - ci-operator
+      env:
+      - name: RELEASE_IMAGE_LATEST
+        value: registry.svc.ci.openshift.org/ocp/release:4.5-ci
+      - name: BRANCH
+        value: "4.5"
+      - name: CLUSTER_TYPE
+        value: gcp
+      - name: JOB_NAME_SAFE
+        value: libvirt-31-day
+      - name: CONFIG_SPEC
+        value: |
+          resources:
+            '*':
+              limits:
+                memory: 4Gi
+              requests:
+                cpu: 100m
+                memory: 200Mi
+          tag_specification:
+            name: "$(BRANCH)"
+            namespace: ocp
+          tests: # this is only here to satisfy config spec validation, it doesn't run
+          - as: e2e-gcp
+            commands: "test"
+            openshift_installer:
+              cluster_profile: gcp
+      - name: TEST_COMMAND
+        value: "cat >test_script.sh <<EOTESTS\n#!/bin/bash\nsudo systemctl stop chronyd\nfuture=\\$(date
```

Comment:
why not add the lines directly with
```
value: |
  sudo systemctl stop chronyd
  future=\\$(date -d \"+31 days\" +%Y-%m-%d)
```

Reply:
i have no idea why I did that, but I've updated to be direct now